tests/compiler/array/length.leo

```diff
@@ -0,0 +1,12 @@
+/*
+namespace: Compile
+expectation: Pass
+input_file: input/length.in
+*/
+
+function main(arr1: [u8; 2], const arr2: [u8; 2]) -> bool {
+    const len1 = arr1.len();
+	const len2 = arr2.len();
+
+	return len1 == len2; 
+}
```

Comment:
Remember to set your editor to always insert a newline at EOF (and then there are 3 files that need fixing in this PR).

Reply:
+1 on that.